Sming/Wiring/FakePgmSpace.cpp

```diff
@@ -1,4 +1,4 @@
-
+#include "WiringFrameworkDependencies.h"
```

Comment:
I am just curious: what is the reason for moving that include from the header file to here?